MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -54,6 +78,16 @@ final class MovieQuizViewController: UIViewController, QuestionFactoryDelegate,
         }
     }
     
+    func didLoadDataFromServer() {
+        hideLoadingIndicator()
+        questionFactory?.requestNextQuestion()
+        
```

Comment:
Ты можешь дополнительно показывать activityIndicator между загрузкой вопросов, потому что после загрузки фильмов, загрузка постера тоже занимает некоторое время из-за получения изображения.
```suggestion
        questionFactory?.requestNextQuestion()
```